modules/helfi_eu_cookie_compliance/config/rewrite/eu_cookie_compliance.settings.yml

```diff
@@ -52,7 +53,7 @@ better_support_for_screen_readers: false
 method: categories
 disabled_javascripts: ''
 automatic_cookies_removal: true
-allowed_cookies: "essential:AWSELB\r\nessential:AWSELBCORS\r\nessential:cookiehub\r\nessential:mtm_cookie_consent\r\nessential:frosmo_quickContext\r\nessential:id\r\nessential:JSESSIONID\r\nessential:COOKIE_SUPPORT\r\npreference:httpskartta.hel.fi.SWCulture\r\npreference:icareus-device\r\npreference:VISITOR_INFO1_LIVE\r\npreference:CONSENT\r\nstatistics:Snoobisession_hel_fi\r\nstatistics:Snoobi30minute_hel_fi\r\nstatistics:SnoobiID\r\nstatistics:nmstat\r\nstatistics:Snoobisession_hel_projektit\r\nstatistics:Snoobi30minute_hel_projektit\r\nstatistics:_pk_id.*\r\nstatistics:_pk_ses.141.89f6\r\nstatistics:_pk_id.*\r\nstatistics:_pk_ses.*\r\nstatistics:_pk_id.*\r\nstatistics:_pk_ses.*\r\nstatistics:__utma\r\nstatistics:__utmc\r\nstatistics:__utmz\r\nstatistics:__utmt\r\nstatistics:__utmb\r\nstatistics:__utmb\r\nstatistics:YSC\r\nmarketing:IDE\r\nchat:_genesys.widgets.*\r\nchat:__VCAP_ID__\r\nchat:gcReturnSessionId \r\nchat:gcAlternativeReturnUrl \r\nchat:gcReturnUrl \r\nchat:gcLoginButtonState\r\nchat:leijuke.*\r\nchat:connect.sid"
+allowed_cookies: "essential:AWSELBCORS\r\nessential:cookiehub\r\nessential:mtm_cookie_consent\r\nessential:JSESSIONID\r\nessential:COOKIE_SUPPORT\r\nessential:GUEST_LANGUAGE_ID\r\npreference:httpskartta.hel.fi.SWCulture\r\npreference:icareus-device\r\npreference:VISITOR_INFO1_LIVE\r\npreference:CONSENT\r\nstatistics:nmstat\r\nstatistics:_pk_id.*\r\nstatistics:_pk_ses.141.89f6\r\nstatistics:_pk_id.*\r\nstatistics:_pk_ses.*\r\nstatistics:_pk_id.*\r\nstatistics:_pk_ses.*\r\nstatistics:YSC\r\nchat:_genesys.widgets.*\r\nchat:leijuke.*"
```

Comment:
This list is missing the following statistics cookies: 
```
rnsbid (askem.com)
rnsbid_ts (askem.com)
rns_reaction_* (askem.com)
```
Is it intentional or a mistake?